### DISCUSSION TITLE
Best way to get newline at end of compiled files and source maps

### DISCUSSION THREAD

**Discussion by kamalx** [glenn2223/vscode-live-sass-compiler#281] [Q&A]
Hello, I just want the generated files to have newlines at the end of file and can't get it to work (the source scss files end with a newline). I have tried various combinations of the `linefeed` setting to no avail -- this also gives a *soon to be removed in Sass v2.0* warning. Is there currently a way to get EOF newline in compiled files through extension settings? Thank you!

**Comment by glenn2223** [comment]
You can't from the extension settings itself.

Maybe you can tie up a file watcher/linter (or create something with postcss) that can append the newlines. You could always recommend it over at sass/dart-sass too, though it is unlikely that they will introduce it.

**Reply by kamalx** [reply]
Thank you for your reponse, Glen.  I'll update here if I manage to find something to achieve this.
